src/librustdoc/html/static/main.js

```diff
@@ -685,7 +685,7 @@
         }
 
         function escape(content) {
-            let h1 = document.createElement('h1');
```

Comment:
Opera 12 is from 2012, so that makes sense. 😅 I don't mind making this change, as it's pretty small, but it is very unlikely that we won't end up doing something that breaks it in the future.